plugins/notifications/ClickAppNotifications.qml

```diff
@@ -42,124 +45,143 @@ ItemPage {
 
     Component.onDestruction: disableNotificationsWhenAllUnchecked()
 
-    Column {
-        id: notificationsColumn
-
-        anchors.fill: parent
-
-        ListItem {
-            height: enableNotificationsLayout.height + (divider.visible ? divider.height : 0)
-            ListItemLayout {
-                id: enableNotificationsLayout
-                title.text: i18n.tr("Notifications")
-                Switch {
-                    id: enableNotificationsSwitch
-                    objectName: "enableNotificationsSwitch"
-                    SlotsLayout.position: SlotsLayout.Trailing
-                    checked: entry ? entry.enableNotifications : false
-
-                    onCheckedChanged: {
-                        ClickApplicationsModel.setNotifyEnabled(ClickApplicationsModel.EnableNotifications,
-                                                                appNotificationsPage.entryIndex,
-                                                                checked)
+    Flickable {
+        id: clickAppNotificationsFlickable
+        flickableDirection: Flickable.VerticalFlick
+
+        anchors {
+            top: pageHeader.bottom
```

Comment:
This should not be needed, the `Page` element will handle it itself. Just use `anchors.fill: parent`.